docker-compose.yml

```diff
@@ -86,7 +89,7 @@ services:
     ports:
       - "8012:8012"
     environment:
-      PUBLIC_PATH: http://${MITX_ONLINE_HOSTNAME:-mitxonline.odl.local}:8012/
+      PUBLIC_PATH: ${MITX_ONLINE_WATCH_BASE_URL:-http://mitxonline.odl.local:8012}
```

Comment:
This (and line 134) need a trailing slash on the default. As-is, the JS app doesn't load.